### DISCUSSION TITLE
personal-summary/cse/在海大的个人成长——避雷这些事-n0vic3

### DISCUSSION THREAD

**Discussion by giscus** [Hainanu-Application/Hainanu-Application.github.io#20] [Announcements]
# /personal-summary/cse/%E5%9C%A8%E6%B5%B7%E5%A4%A7%E7%9A%84%E4%B8%AA%E4%BA%BA%E6%88%90%E9%95%BF%E2%80%94%E2%80%94%E9%81%BF%E9%9B%B7%E8%BF%99%E4%BA%9B%E4%BA%8B-n0vic3

A magical documentation generator.

https://hainanu-application.github.io/#/personal-summary/cse/%E5%9C%A8%E6%B5%B7%E5%A4%A7%E7%9A%84%E4%B8%AA%E4%BA%BA%E6%88%90%E9%95%BF%E2%80%94%E2%80%94%E9%81%BF%E9%9B%B7%E8%BF%99%E4%BA%9B%E4%BA%8B-n0vic3

**Comment by Threesirui** [comment]
你需要介绍一下你自己，多点说服力

**Comment by dy033** [comment]
确实是这样，哎，可惜看的晚了，已经大三升大四了

**Reply by peng-yq** [reply]
正因为如此，我们创建了这个平台，学弟可以贡献呀。

**Comment by ColdJeans21** [comment]
我也看晚了，悲😫
